docs/contributors/github-labels.md

```diff
@@ -56,19 +56,22 @@ The labels in this list originated within Kubernetes at
 | area/provider/azure                | Issues or PRs related to azure provider | Any |
 | area/provider/gcp                  | Issues or PRs related to gcp provider | Any |
 | area/provider/vmware               | Issues or PRs related to vmware provider | Any |
-| area/routing                       | Issues or PRs related to routing and forwarding | Any |
+| area/proxy                         | Issues or PRs related to proxy functions in Antrea | Any |
+| area/proxy/clusterip               | Issues or PRs related to the implementation of ClusterIP Services | Any |
+| area/proxy/nodeport                | Issues or PRs related to the implementation of NodePort Services | Any |
+| area/proxy/nodeportlocal           | Issues or PRs related to the NodePortLocal feature | Any |
```

Comment:
Feel nodeportlocal is not really proxy, but probably ok if we do not have a better category for it.

Reply:
do you think I should rename the "proxy" category to "services"

Reply:
Humm. It might be easier for users do not understand AntreaProxy, but for nodeportlocal maybe later we can apply it to any Pods selected by a selector or group?

Reply:
Makes sense. I'll probably stick to "proxy" then. I think it's still relatively accurate even for NPL. I like to think of it as similar to what kube-proxy does for a NodePort Service with `externalTrafficPolicy` set to `local`.

Reply:
Sounds good.